pkgs/applications/audio/youtube-music/default.nix

```diff
@@ -93,6 +137,6 @@ buildNpmPackage {
     license = licenses.mit;
     maintainers = [ maintainers.aacebedo ];
     mainProgram = "youtube-music";
-    platforms = platforms.all;
+    platforms = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

Comment:
This seems more wrong than `platforms.all`, as `electron` supports more Linux platforms.

Reply:
I used the one that was build. The value is not so important to me, I just want to get this merged.

Reply:
> The value is not so important to me, I just want to get this merged.

It could be for users of exotic platforms. I'll suggest making the (personally blocking) change in another PR.

Reply:
All right, then. Good luck to users of exotic platforms.